decidim-core/app/views/decidim/account/show.html.erb

```diff
@@ -2,6 +2,9 @@
   <%= decidim_form_for(@account, url: account_path, method: :put, class: "user-form") do |f| %>
     <div class="columns large-4">
       <%= f.upload :avatar %>
+      <% if @account.errors[:avatar].any? %>
+        <p class="is-invalid-label"><%= safe_join @account.errors[:avatar], '<br/>' %></p>
```

Comment:
Maybe we should move this to the `upload` method of the FormBuilder, so other upload fields can benefit from this. Do you want to give it a try? 😄 

(note that since `form_builder.rb` file is in a `lib/` directory you'll need to restart your server for every change :( )